jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -0,0 +1,100 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    environment {
+        AWS_ROLE_ARN = "arn:aws:iam::${AWS_ACCOUNT_PUBLIC}:role/opensearch-test"
+        AWS_ROLE_SESSION_NAME = "jenkins-test-session"
```

Comment:
Where are `AWS_ROLE_SESSION_NAME` and `AWS_ROLE_ARN` variables used? 
Similar question for the parameters

Reply:
Removed unused variables.